SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/ManagedWorkerCoordinator.cs

```diff
@@ -33,6 +34,9 @@ internal class ManagedWorkerCoordinator : AbstractWorkerCoordinator
 
         private const string CoordinatorWorkerType = "SimulatedPlayerCoordinator";
         private const string SimulatedPlayerFilename = "StartSimulatedClient.sh";
+        private const string StopSimulatedPlayerFilename = "StopSimulatedClient.sh";
+
+        private const int MillisToTicks = 10000;
```

Comment:
There is `TimeSpan.TicksPerMillisecond`, I'd prefer this one since it's provided by the language.

Reply:
Cool, I will update it.